x-pack/plugins/observability_solution/slo/server/routes/slo/route.ts

```diff
@@ -598,11 +617,13 @@ const fetchSloHealthRoute = createSloServerRoute({
   handler: async ({ context, params, logger }) => {
     await assertPlatinumLicense(context);
 
-    const soClient = (await context.core).savedObjects.client;
-    const esClient = (await context.core).elasticsearch.client.asCurrentUser;
+    const core = await context.core;
+    const scopedClusterClient = core.elasticsearch.client;
+    const soClient = core.savedObjects.client;
+    const esClient = core.elasticsearch.client.asCurrentUser;
     const repository = new KibanaSavedObjectsSLORepository(soClient, logger);
 
-    const getSLOHealth = new GetSLOHealth(esClient, repository);
+    const getSLOHealth = new GetSLOHealth(esClient, scopedClusterClient, repository);
```

Comment:
@dominiqueclarke I was checking all the routes in this file and I was wondering if we need to pass the `scopedClusterClient` to the `getSloBurnRates` route as well. I noticed that you didn't change the GET requests, but getSloBurnRates is `POST` request, so I thought we might need to pass the scopedClusterClient there as well. What do you think?

Reply:
There aren't any transform or ingest pipeline actions in that code path.